client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -48,6 +65,21 @@ const BillingIntervalSwitcher: React.FunctionComponent< Props > = ( {
 						onClick={ () => onChange( IntervalLength.ANNUALLY ) }
 					>
 						{ annualLabel }
+						{ plugin ? (
```

Comment:
Can we check for `plugin?.saving` her or something so we don't have to do multiple checks?

Not sure if the `&&` notation would work here instead of the ternary?
`{ plugin && ( <PluginAnnualSaving /> ) }`

Same below.

Reply:
I fixed it in [this commit](https://github.com/Automattic/wp-calypso/pull/61341/commits/35806f3d082ede6f7e5577351afd86440f1334d6)